### ISSUE TITLE
bug: Overlapping views when navigating quickly

### ISSUE THREAD

**Issue by tripleneo** [driftyco/ionic#4499]
**Type**: <span ionic-type>bug</span>

**Platform**: <span ionic-platform>ios</span> <span ionic-platform-version>8</span>  <span ionic-webview>webview</span>

<span ionic-description>I have three views: groups, activiteis of a group and activity detail view. When I quickly browse between views in some cases I end up with two overlapping views when browsing back. Both views are showing 50% transparent. I am using the AngularUIRouter with states and a resolve part to load the detailed activity. When I disable view caching it gets better. Still you see overlapping views in a flash but then the view that should display is put in front of the other view and it looks normal again. 
To summarize: the app behaves normal first, but after a while and when tapping quickly, multiple times on the same nav links, this issue occurs. And it occurs only when browsin backwards, not forwards.</span>

<span is-issue-template></span>

**Comment by kdubuc** [comment]
I can see the flickering too, and in my case I think #4493 is related

**Comment by dmarg** [comment]
+1 I am experiencing a similar problem on a project at the moment. 

Seems like the transition back to a view hangs and the two views are 50% transparent.

**Comment by lastmjs** [comment]
I think I'm experiencing the same issues, Android 6.0, Nexus 5, Crosswalk, and on Ubuntu 14.04 Chrome desktop.

**Comment by lastmjs** [comment]
@tripleneo What version of Ionic are you experiencing the issues on?

**Comment by jgw96** [comment]
Hello! Are you guys still experiencing this issue with the latest version of both Ionic 1 and Ionic 2?

**Comment by jgw96** [comment]
Hello! Thanks for opening an issue with us! Since it has been a little while since there was any activity on this issue i will be closing it, but feel free to reopen this issue if needed!
